byway/src/main/java/com/google/sps/servlets/DestinationsServlet.java

```diff
@@ -40,10 +39,10 @@ public void init() {
     datastore.put(userEntity);
     userKey= userEntity.getKey();
   }
-
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Entity entity; 
+    Entity entity = null; 
     try{ 
       entity = datastore.get(userKey);
     } catch(EntityNotFoundException e){
```

Comment:
I think lines 55-59 could be inside the try statement rather than outside to reduce the redundancy. For instance, notice that `response.setContentType(...)` and `response.getWriter().println(...)` can be outside of try/catch, and the try/catch statements can handle getting/making a `UserLocation` object.

Reply:
I had that before, but zach told me to limit the scope of the catch block to avoid errors that weren't caused by the datastore.get